packages/core/index.js

```diff
@@ -111,6 +111,8 @@ const clearStyles = () => {
 
 const RESHADOW_ID = '__reshadow__';
 
+const appendChildToDocumentHead = element => document.head.appendChild(element);
```

Comment:
```suggestion
const appendChildToDocumentHead = document.head.appendChild;
```

Reply:
that will also cause an error, because there is a problem of access to the `document.head` before the DOM initialization

Reply:
@lttb, are you want to put `document.addEventListener` to this function?